editor/index.js

```diff
@@ -68,5 +68,6 @@ export function createEditorInstance( id, post, settings ) {
 		initializeMetaBoxes( metaBoxes ) {
 			provider.store.dispatch( initializeMetaBoxState( metaBoxes ) );
 		},
+		...provider.hooks,
```

Comment:
Do you think we could avoid having to register the hooks to the editor's instance. What if we have multiple instances?. I'm thinking a global like `wp.postEditor.hooks` could be better.

Reply:
>Do you think we could avoid having to register the hooks to the editor's instance. What if we have multiple instances?. I'm thinking a global like `wp.postEditor.hooks` could be better. 

Right, this is a bigger open question about hooks generally. It seems in this case we have a few options:

1. A single hooks registry, e.g. `wp.hooks.doAction`
2. A hooks registry common to all editors, e.g. `wp.postEditor.doAction`
3. A hooks registry specific to a single instance of an editor, e.g. `myEditor.doAction`

On the last two, it depends if we think it would be common for multiple instances of an editor to exist in the same screen and, if so, whether it's sensible that each have distinct or share common extension behaviors. And generally whether we think the scoping adds value (clarity? predictable flow?)

It's not too dissimilar to as in Redux, we could have a module exporting a global shared instance of the store (`import store from '../store';`) rather than passing it around as an argument when initialized (usually through the React [Redux Provider context](https://github.com/reactjs/react-redux/blob/master/docs/api.md#provider-store)).